jaxrs/src/main/java/se/fortnox/reactivewizard/jaxrs/RequestLogger.java

```diff
@@ -57,10 +63,23 @@ public static void headersToString(HttpServerResponse response, StringBuilder lo
      * @return value of the header or "REDACTED" if the header contains sensitive information
      */
     public static String getHeaderValueOrRedact(Map.Entry<String, String> header) {
+        return getHeaderValueOrRedact(header, emptyList());
+    }
+
+    /**
+     * Returns the value of a header. If the header contains sensitive information the value will be replaced with "REDACTED".
+     *
+     * @param header           The header to get the value from
+     * @param sensitiveHeaders Sensitive headers to redact
+     * @return value of the header or "REDACTED" if the header contains sensitive information
+     */
+    public static String getHeaderValueOrRedact(Map.Entry<String, String> header, List<String> sensitiveHeaders) {
         if (header == null) {
             return null;
-        } else if ("Authorization".equalsIgnoreCase(header.getKey())) {
-            return "REDACTED";
+        } else if (AUTHORIZATION_HEADER.equalsIgnoreCase(header.getKey())) {
+            return REDACTED;
+        } else if (sensitiveHeaders.stream().anyMatch(header.getKey()::equalsIgnoreCase)) {
```

Comment:
This will run on every header of every log, so would it be possible to have a `Set` instead so that we get fast lookups here?

Reply:
Good idea, I'm on it :)